tools/hifiasm/hifiasm.xml

```diff
@@ -428,9 +454,12 @@
             <param name="filter_bits" value="0"/>
             <conditional name="mode">
                 <param name="mode_selector" value="trio"/>
-                <param name="reads" value="child.fasta.gz"/>
-                <param name="hap1_reads" value="paternal.fasta.gz"/>
-                <param name="hap2_reads" value="maternal.fasta.gz"/>
+                <conditional name="trioinput">
+                    <param name="reads" value="child.fasta.gz"/>
```

Comment:
reads needs to be in the outer / one level above conditional?

Reply:
yeah, so that's what i thought too when i was initially fixing test 5 LOL but then it didn't work that way, so i made it this way... and now it works inside the conditional in test 5, but not in test 15. but in test 15, it DOES work outside the conditional: 

<img width="2230" alt="image" src="https://github.com/bgruening/galaxytools/assets/48773001/9967d0de-c004-4b8f-a67e-2f4ddad7d44b">

Reply:
(both views are the same file just at different positions)

Reply:
let me add in your suggestions though!! 👍🏼